src/features/footer/constants.js

```diff
@@ -0,0 +1,7 @@
+export const EVENT_NAMES = {
+  ADA_LINK: 'edx.ada.footer.link',
+  PRIVACY_POLICY_LINK: 'edx.privacy.policy.footer.link',
+  TERMS_OF_SERVICE_LINK: 'edx.terms.of.service.footer.link',
+};
+
+export const copySymbol = String.fromCodePoint(0x00A9);
```

Comment:
because this is a constant, I think it would be better to name it un uppercase like COPY_SYMBOL